File: 프로그래머스/2/12941. 최솟값 만들기/최솟값 만들기.js
function solution(A,B){
    var ans = 0;
    A.sort((a, b) => {return a - b})
    B.sort((a, b) => {return b - a})
    for (let i = 0; i < A.length; i++)
        ans += A[i] * B[i]
    return ans;
}

File: 프로그래머스/2/12941. 최솟값 만들기/최솟값 만들기.py
def solution(A,B):
    n = len(A)
    
    # 큰 값  * 작은 값 누적 합 = 최소
    A.sort()  # 오름차순 정렬
    B.sort(reverse=True)  # 내림차순 정렬
    
    return sum(A[i] * B[i] for i in range(n))